README.md

```diff
@@ -0,0 +1,52 @@
+# 🐍 AI (Actual Intellgence)  
+
+Welcome to the **Actual Intellgence (AI) group**!
+This group is a space where our group collaborates to solve fun and
+challenging Python problems while enhancing our coding skills.  
+
+## 👥 Meet the Team  
+
+Our group is a blend of creative minds and problem-solving enthusiasts.
+Here’s a quick introduction to everyone, along with a fun fact about them:  
+
+- **Karim** ⚽  
+  Fun Fact: I probably spend way too much time watching football and basketball!
+
+- **Nico** 🎞️📺  
+  Fun Fact: I love tv shows. My favs are modern family and Brooklyn nine-nine.
+
+- **Marshar** 🍕  
+  Fun Fact: I can die for food, can't for assignment.
```

Comment:
For better grammar, it can be changed to: 'I can die for food but not for assignments'.